farm.coffee

```diff
@@ -59,22 +59,26 @@ module.exports = exports = (argv) ->
         .map (item) -> item.trim()      # trim any whitespace padding from the items in the list
 
     allowHost = (host) ->
-      hostDomain = _.split(host, ':')[0]
-      if _.includes(allowedHosts, hostDomain)
+      hostDomain = host.split(':')[0]
+      if allowedHosts.includes(hostDomain)
         return true
       else
         return false
 
   if argv.wikiDomains
-    wikiDomains = _.keys(argv.wikiDomains)
+    wikiDomains = Object.keys(argv.wikiDomains)
     inWikiDomain = ''
     allowDomain = (host) ->
-      hostDomain = _.split(host, ':')[0]
-      inWikiDomain = ''
-      _.each wikiDomains, (domain) ->
-        if _.endsWith hostDomain, domain
-          inWikiDomain = domain
-      if inWikiDomain
+      hostDomain = host.split(':')[0]
+      possibleWikiDomain = []
+      wikiDomains.forEach((domain) ->
+        if hostDomain.endsWith(domain)
+          possibleWikiDomain.push(domain)
+        )
+      if possibleWikiDomain
```

Comment:
I think `possibleWikiDomain` will always be truthy 'cos line 73 assigns an empty array. Suggest:
```
if possibleWikiDomain.length >= 0
```

Reply:
Now checking that `length > 0`.